src/harmony/Checkbox/Checkbox.module.css

```diff
@@ -1,9 +1,9 @@
 .Checkbox {
     position: relative;
     display: inline-flex;
-    align-items: center;
     cursor: pointer;
     color: var(--text-primary);
+    font-size: var(--font-size-m);
```

Comment:
why not inherited?

Reply:
inherit from? however Checkbox haven't sizes props